server/app/permissions.go

```diff
@@ -342,21 +342,24 @@ func PlaybookModify(userID string, playbook, oldPlaybook Playbook, cfgService co
 	return nil
 }
 
-func ModifySettings(userID string, config config.Service) error {
+func ModifySettings(userID string, isAdmin bool, config config.Service) error {
```

Comment:
Maybe we should rename this to be explicit given that this behavior is specifically for that one setting. 
```suggestion
func ModifyPlaybookCreators(userID string, isAdmin bool, config config.Service) error {
```